dockstore-client/src/main/java/io/github/collaboratory/cwl/cwlrunner/ToilWrapper.java

```diff
@@ -49,9 +51,15 @@ public List<String> getExecutionCommand(String outputDir, String tmpDir, String
         //TODO: this doesn't quite work yet, seeing "toil.batchSystems.abstractBatchSystem.InsufficientSystemResources: Requesting more disk
         // than either physically available, or enforced by --maxDisk. Requested: 537944653824, Available: 134853001216" on trivial
         // workflows like md5sum
+
+        //try normalizing paths
+        Path currentRelativePath = Paths.get("");
+        Path tmpPath = Paths.get(currentRelativePath.toAbsolutePath().toString(), tmpDir);
```

Comment:
Apparently, `Paths` may be deprecated in a future release, so probably better to use `Path.of` per https://stackoverflow.com/questions/4871051/how-to-get-the-current-working-directory-in-java